Tree/readme.md

```diff
@@ -47,6 +47,7 @@ that, this property should be satisfied at every node in the tree.
 * Sorting the elements of binary search tree and many more
 
 ## Questions :
+* Searching in BST ----> [C++](/Code/C++/searching_in_bst.cpp) 
 * Check if a Binary Tree is a valid Binary Search Tree ----> [Java](/Code/Java/check_valid_BST.java)
 * Searching in BST ----> [C++](/Code/C++/searching_in_bst.cpp)
```

Comment:
see it is already there... the line you added

Reply:
Done